src/projects/EnsureThat/ExceptionMessages.cs

```diff
@@ -28,6 +28,8 @@ public static class ExceptionMessages
         public static string Strings_SizeIs_Failed { get; } = "Expected length '{0}' but found '{1}'.";
         public static string Strings_IsNotNullOrWhiteSpace_Failed { get; } = "The string can't be left empty, null or consist of only whitespaces.";
         public static string Strings_IsNotNullOrEmpty_Failed { get; } = "The string can't be null or empty.";
+
+        public static string Strings_IsEmptyOrWhitespace_Failed { get; } = "The string can't be empty or Whitespace only (but can be null)";
```

Comment:
Probably shouldn't capitalize Whitespace. It's also 2 words. Finally, may want to include an 'it' for the subject in the parenthetical clause.
e.g. "The string can't be empty or only white space (but it can be null)".

Alternatively I'd suggest language promoting the can-be-null clause out of the parenthetical like "When the string is supplied (not null), it can't be empty or only white space"

Reply:
Fixed (approximately)